app/views/streams/main_stream.html.haml

```diff
@@ -31,8 +31,13 @@
       %ul#stream_selection
         %li.hoverable{:data => {:stream => 'stream'}}
           = link_to t("streams.multi.title"), stream_path, :rel => 'backbone'
-        %li.hoverable{:data => {:stream => 'activity'}}
-          = link_to t("streams.activity.title"), activity_stream_path, :rel => 'backbone'
+        %li{:data => {:stream => 'activity'}}
+          = link_to t("streams.activity.title"), activity_stream_path, :rel => 'backbone', :class => 'hoverable'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.